### DISCUSSION TITLE
Customize Admin Lte Settings

### DISCUSSION THREAD

**Discussion by sanganitarvik** [ColorlibHQ/AdminLTE#3333] [Q&A] [answered]
**Describe the bug**
When Page Refreshed , Customize Admin Lte settings automatically set to default 

**To Reproduce**
1. Go to Customize Admin Lte settings (Top Right Corner)
2. Click on any Color of Skins
3. Now Refresh Page

**Expected behavior**
When Customize Settings Changed , Should be set until clear cache data.

**Environment (please complete the following information):**
- AdminLTE
- Browser (Version): [e.g. Chrome (Latest)]

**Comment by danny007in** [comment, marked as answer]
That is only for demo purpose u can check in https://github.com/ColorlibHQ/AdminLTE/blob/master/dist/js/demo.js
